### DISCUSSION TITLE
Does MuJoCo have a Visual Studio build?

### DISCUSSION THREAD

**Discussion by yzd0014** [google-deepmind/mujoco#429] [Asking for Help] [answered]
I wonder if MuJoCo supports CMake to generate a Visual Studio solution file so that I can work with source code of MuJoCo in VS instead of just the static library. Thank you in advance!

**Comment by yzd0014** [comment, marked as answer]
I think I found an answer. I can just use Cmake to generate a VS solution file, and  it works pretty well.

**Reply by saran-t** [reply]
If you'd like contribute a short section in the documentation on how to do this that'd be much appreciated. :)
